tests/optimization/test_pounders_integration.py

```diff
@@ -40,12 +42,12 @@ def func(x: np.ndarray, exog: np.ndarray, endog: np.ndarray) -> np.ndarray:
 
 start_params = [
     np.array([0.15, 0.008, 0.01]),
-    np.ones(3) * 0.25,
+    # np.ones(3) * 0.25, # noqa: E800
```

Comment:
```suggestion
```

Reply:
I deleted the line ```# np.ones(3) * 0.25, # noqa: E800``` or do you want to keep it for future reference?